### PR TITLE
FIX-1068: reverse custom loop

### PR DESCRIPTION
**Pull request by DenisYaroshevskiy**
Let's not merge until I measure.

See what are we doing in the issue: https://github.com/jfalcou/eve/issues/1068

### CODE REVIEW

File: include/eve/algo/transform.hpp [hunk 1]
```diff
@@ -36,7 +36,7 @@ namespace eve::algo
         using out_t = eve::element_type_t<decltype(processed)>;
         auto cvt_and_store_it = views::convert(store_it, as<out_t>{});
 
-        eve::store[ignore](op(xs), cvt_and_store_it);
+        eve::store[ignore](processed, cvt_and_store_it);
```
Comment (jfalcou) [inline include/eve/algo/transform.hpp:39]: why this change ?

Comment (DenisYaroshevskiy) [reply include/eve/algo/transform.hpp:39]: Because I called `op` twice - now just call once.

**Review by jfalcou — APPROVED** [review]
